### PR TITLE
Added virtual paint

### LINKED ISSUE #519: Virtual Paint
# Aim

- Detecting Color and using colored marker to draw virtually

# Details

- It is an OpenCV application that can track an object’s movement, using which a user can draw on the screen by moving the object around — I call it Webcam Paint.
- I will implement it with :
Step 1: Initialize some stuff (import the necessary libraries)
Step 2: Start reading the video (frame by frame)
Step 3: Find the Contour-Of-Interest
Step 4: Start drawing and store the drawings
Step 5: Show the drawings on the screen

- It will look like this:

virtual_paint

### PR DESCRIPTION
**Pull request by mehabhalodiya**
# Description

It is an OpenCV application that can track an object’s movement, using which a user can draw on the screen by moving the object around — I call it Webcam Paint.

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

_Please delete options that are not relevant._

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings
- [ ] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

**Review by kaustubhgupta — APPROVED** [review]
Nice implementation 💯 👍🏼

**Review by antrikshmisri — APPROVED** [review]
@kaustubhgupta LGTM 🚀

File: Virtual Paint/virtual_paint.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    newPoints = findColor(img, myColors, myColorValues)
+    if len(newPoints) != 0:
+        for newP in newPoints:
+            myPoints.append(newP)
+    if len(myPoints) != 0:
+        drawOnCanvas(myPoints, myColorValues)
+
+    cv2.imshow("Result", imgResult)
+    if cv2.waitKey(1) & 0xFF == ord('q'):
+        break
```
Comment (santushtisharma10) [inline Virtual Paint/virtual_paint.py:71]:
```suggestion
        break
        
cap.release()
cv2.destroyAllWindows()
```

Comment (santushtisharma10) [reply Virtual Paint/virtual_paint.py:71]: @mehabhalodiya you should add these two lines of code at the end of your code.

Comment (mehabhalodiya) [reply Virtual Paint/virtual_paint.py:71]: Added!

**Review by santushtisharma10 — APPROVED** [review]

**Review by avinashkranjan — APPROVED** [review]

**Pull request merged**